packages/core/src/tracing/cls.js

```diff
@@ -565,6 +568,11 @@ function skipExitTracing(options) {
     return true;
   }
 
+  if (allowRootExitSpan) {
```

Comment:
Only this check is required as the suppresed check is already covered in the above line. If tracing is suppressed then there is no point in sending spans according to `allowRootExitSpan`